Parquet/ResultListener/ParquetFile.cs

```diff
@@ -28,16 +30,26 @@ internal sealed class ParquetFile : IDisposable
 
         public Schema Schema { get; }
 
-        internal ParquetFile(Schema schema, Stream writeStream, ParquetFileOptions? options = null)
+        internal ParquetFile(Type generatingTool, Schema schema, Stream writeStream, ParquetFileOptions? options = null)
         {
             Schema = schema;
             _stream = writeStream;
             _writer = new ParquetWriter(Schema, _stream);
             _dataCache = schema.GetDataFields().ToDictionary(field => field, field => new ArrayList());
             Options = options ?? new ParquetFileOptions();
+
+            string assemblyLocation = generatingTool.Assembly.Location;
+            FileVersionInfo versionInfo = FileVersionInfo.GetVersionInfo(assemblyLocation);
+            _writer.CustomMetadata = new Dictionary<string, string>()
+            {
+                {"SchemaVersion", new Version(1, 0, 0, 0).ToString() },
+                {"Tool", generatingTool.FullName },
+                {"Time", DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss") },
```

Comment:
Please provide feedback on if this format is the correct format, or if there might be a different format to use.

Reply:
This is not the best format, it requires the user to know which timezone the measurement took place in in order to understand the value. Use Now.ToString("o", CultireInfo.InvariantCulture) instead.